tests/unit/utils/test_utils.py

```diff
@@ -926,6 +926,21 @@ def test_system_default_application(self, caplog, config_stub, mocker):
 def test_unused():
     utils.unused(None)
 
+@pytest.mark.parametrize('txtset, nummax, expected', [
+    ('0-5,9-100,last', 500, [(0, 5), (9, 100), (500, 500)]),
+    ('0-5,9-100,last-3', 500, [(0, 5), (9, 100), (500, 3)]),
+    ('0-5,9-100,last-7000', 500, [(0, 5), (9, 100), (500, 7000)]),
```

Comment:
My latest comment has my 1st attempt to test for the last one (malformed input).